### ISSUE TITLE
Deep links not opening correctly

### ISSUE THREAD

**Issue by trygve55** [IsmaelMartinez/teams-for-linux#1384]
**Describe the bug**
Deep links (`msteams://`) for meeting invites does not work.

**To Reproduce**
1. Create a new meeting.
2. Right click on this meeting and choose `Copy link`
3. Open an terminal and run `teams-for-linux msteams:<copied URL without https:>`
4. Teams for Linux will start loading and show a message "Setting things up for you..."
5. It finishes loading and nothing happens.

**Expected behavior**
The meeting will open as normaly.

**Desktop:**
 - OS: Ubuntu 22.04 LTS
 - Installation package: snap
 - Version: 1.9.1

**Debug**
When possible, please run the application from the terminal using `--webDebug` and try to reproduce the error.

The provide in this section the output from both the terminal and the browser debug console.

First terminal command:
```bash
teams-for-linux --webDebug --logConfig='{}'
```

Second terminal command
```bash
teams-for-linux [--webDebug --logConfig='{}'](msteams://teams.microsoft.com/l/meetup-join/****?context=****)
```

Output (first terminal):
```
14:21:09.502 › second-instance started
14:21:09.502 › processArgs: [
  '/snap/teams-for-linux/658/teams-for-linux',
  '--no-sandbox',
  '--allow-file-access-from-files',
  '--try-supported-channel-layouts',
  '--disable-features=HardwareMediaKeyHandling',
  '--auth-server-whitelist=*',
  '--enable-ntlm-v2=true',
  'msteams://teams.microsoft.com/l/meetup-join/****?context=****'
]
14:21:09.503 › testing RegExp processArgs false
14:21:09.504 › testing RegExp processArgs false
14:21:09.504 › testing RegExp processArgs false
14:21:09.504 › testing RegExp processArgs false
14:21:09.504 › testing RegExp processArgs false
14:21:09.504 › testing RegExp processArgs false
14:21:09.504 › testing RegExp processArgs false
14:21:09.504 › testing RegExp processArgs false
14:21:09.504 › A url argument received with msteams protocol
14:21:10.005 › GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 0s, IdleState: 'active'
14:21:10.873 › did-frame-finish-load {} true
14:21:10.874 › did-finish-load
14:21:13.396 › Badge count set to '0'
14:21:13.427 › Badge count set to '0'
14:21:15.735 › Badge count set to '0'
14:21:17.982 › did-frame-finish-load {} false
[970273:0828/142117.990386:ERROR:CONSOLE(1)] "Request Autofill.enable failed. {"code":-32601,"message":"'Autofill.enable' wasn't found"}", source: devtools://devtools/bundled/core/protocol_client/protocol_client.js (1)
14:21:20.064 › GetSystemIdleState => IdleTimeout: 300s, IdleTimeoutPollInterval: 10s, ActiveCheckPollInterval: 2s, IdleTime: 1s, IdleState: 'active'
```

![Screenshot from 2024-08-28 14-27-16](https://github.com/user-attachments/assets/65d6e972-9284-4eb8-9519-d5de657e315b)

**Additional context**

Running the command `teams-for-linux https://teams.microsoft.com/l/meetup-join/****?context=****` works correctly, but `teams-for-linux msteams://teams.microsoft.com/l/meetup-join/****?context=****` produces the bug described above.

**Comment by IsmaelMartinez** [comment]
Can you try the pre-release 1.9.5? I think this is the same as another open issue that should be fixed in 1.9.5. Thanks.

**Comment by trygve55** [comment]
It works in 1.9.5! Thank you for the great work.
